test/async-hooks/test-graph.http.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const common = require('../common');
+const initHooks = require('./init-hooks');
+const verifyGraph = require('./verify-graph');
+const http = require('http');
+
+const hooks = initHooks();
+hooks.enable();
+
+const server = http.createServer(common.mustCall(function(req, res) {
+  res.end();
+  this.close(common.mustCall());
+}));
+server.listen(0, common.mustCall());
+
+http.get(`http://127.0.0.1:${server.address().port}`, common.mustCall());
```

Comment:
should this be moved inside the listening event handler? Wonder if this could possibly end up being flaky